src/AspNetCoreModuleV2/CommonLib/debugutil.h

```diff
@@ -27,23 +83,24 @@ DebugPrint(
     STACK_STRA (strOutput, 256);
     HRESULT  hr = S_OK;
 
-    if ( IfDebug( dwFlag ) )
+    if ( IsEnabled( dwFlag ) )
     {
-        hr = strOutput.SafeSnprintf( 
-            "[aspnetcore.dll] %s\r\n",
-            szString );
+        hr = strOutput.SafeSnprintf(
+            "[%s] %s\r\n",
+            DEBUG_LABEL_VAR, szString );
 
         if (FAILED (hr))
         {
-            goto Finished;
+            return;
         }
 
         OutputDebugStringA( strOutput.QueryStr() );
-    }
-
-Finished:
 
-    return;
+        if (IsEnabled(ASPNETCORE_DEBUG_FLAG_CONSOLE))
```

Comment:
We did not do console log as w3wp does not have a console at all. For developer scenario, writing to debug context is sufficient enough. Instead of writing to console, we should consider writing to ETW log.

Reply:
I want to see logs from functional tests, the easiest way to get them is to log to console.

Reply:
I would also like to be able to get logs into stdout log file so we can diagnose issues when running in deployed environments. But this is a task for the future.